src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final Random random = new Random();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[4];
+        FigureSupplier figure = new FigureSupplier();
+        Figure[] allFigures = new Figure[6];
```

Comment:
```suggestion
```

Reply:
It is array of Figures which I want to get, I need to have this one